tasks/actorstate/miner.go

```diff
@@ -193,6 +193,10 @@ func ExtractMinerInfo(a ActorInfo, ec *MinerStateExtractionContext) (*minermodel
 		ConsensusFaultedElapsed: int64(newInfo.ConsensusFaultElapsed),
 		ControlAddresses:        newCtrlAddresses,
 		MultiAddresses:          newMultiAddrs,
+
+		// ShortString abbreviates the size as a human-scale number.
+		// This approximates (truncates) the size unless it is a power of 1024.
+		SectorSize: newInfo.SectorSize.ShortString(),
```

Comment:
@hsanjuan what format would you prefer `sector_size` to be tracked with? Currently, this string will be one of: `1KiB`, `1MiB`, `1GiB`, `32GiB`, `64GiB`, etc. The other option would be an uint64: `1024`, `1048576`, `1073741824`, `34359738368`, `68719476736`, etc.

Reply:
uint64 please

Reply:
ammended.